### PR TITLE
HTM-531: Add variable to initialize empty config db when starting Docker stack

### PR DESCRIPTION
**Pull request by mvdstruijk**
HTM-531

### CODE REVIEW

**Review by mprins — CHANGES_REQUESTED** [review]
- Where is the reference to the story in Jira? please create as part of https://b3partners.atlassian.net/browse/HTM-10
- Why are the added group descriptions only available in Dutch?

**Review by matthijsln — CHANGES_REQUESTED** [review]
Add documentation in README

**Review by matthijsln — APPROVED** [review]

File: docker/db/docker-entrypoint-initdb.d/3-initialize-empty-database.sql.script [hunk 1]
```diff
@@ -0,0 +1,11 @@
+insert into category (id,name) values(0,'Categoriën');
```
[the remark this answers is not in the document]
Comment (matthijsln) [reply docker/db/docker-entrypoint-initdb.d/3-initialize-empty-database.sql.script:1]: We can think about solving this but not necessarily in this PR, the preloaded dump is not localized anyway. The user can just rename it.

[the remark this answers is not in the document]
Comment (mprins) [reply docker/db/docker-entrypoint-initdb.d/3-initialize-empty-database.sql.script:1]: You could add 2 versions of the empty database 1 English, one Dutch that you can have the admin select using the/an env variable. eg. `CONFIG_DB_INIT_EMPTY_NL` and `CONFIG_DB_INIT_EMPTY_EN`
note it is also and more importantly the user group descriptions, not just `Categoriën`.

but I'm not sure it is worth the effort at this time as the rest of the admin GUI seems to always be in Dutch always

**Review by mprins — APPROVED** [review]

**Pull request merged**